src/main/java/org/umssdiplo/automationv01/core/managepage/smartHouse/SHLogin.java

```diff
@@ -0,0 +1,50 @@
+package org.umssdiplo.automationv01.core.managepage.smartHouse;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+import org.umssdiplo.automationv01.core.utils.PropertyAccessor;
+
+public class SHLogin extends BasePage {
+    @FindBy(name = "username")
+    private WebElement usernameInputField;
+
+    @FindBy(name = "password")
+    private WebElement passwordInputField;
+
+    @FindBy(name = "login_button")
+    private WebElement loginBtn;
+
+    @FindBy(name = "accept_login_button")
+    private WebElement aceptLoginBtn;
+
+    @FindBy(name = "close_login_button")
+    private WebElement closeLoginBtn;
+
+    public void setCredentials() {
+        CommonEvents.clickButton(loginBtn);
+        String username = PropertyAccessor.getInstance().getSHUser();
```

Comment:
Slipt the following events that it necessary to do
Example

public void enterUserFromSystem(){
String username = PropertyAccessor.getInstance().getSHUser();
 CommonEvents.setInputField(usernameInputField, username);
}

maybe you will need to validate with wrong values

public void enterUsername(String username){
 CommonEvents.setInputField(usernameInputField, username);
}